src/Libraries/Microsoft.Extensions.AI/Utilities/AIJsonUtilities.Schema.cs

```diff
@@ -138,8 +138,6 @@ public static JsonElement CreateJsonSchema(
         JsonSerializerOptions? serializerOptions = null,
         AIJsonSchemaCreateOptions? inferenceOptions = null)
     {
-        _ = Throw.IfNull(serializerOptions);
```

Comment:
I'm surprised this slipped through. Are we missing tests for other parameters as well?

Reply:
We were testing null being passed to non-null parameters but not the opposite. I scanned APIs for other occurrences but couldn't find any.